common/scripted_effects/wc_scripted_effects.txt

```diff
@@ -708,73 +708,6 @@ drain_void_source_effect = {
 		}
 	}
 }
-sacrifice_all_prisoners_effect = {
```

Comment:
Still used in the files. You can't access the event though.

Reply:
Removed events